Cargo.toml

```diff
@@ -365,6 +366,7 @@ solana-rpc-client-api = { path = "rpc-client-api", version = "=2.0.0" }
 solana-rpc-client-nonce-utils = { path = "rpc-client-nonce-utils", version = "=2.0.0" }
 solana-runtime = { path = "runtime", version = "=2.0.0" }
 solana-runtime-transaction = { path = "runtime-transaction", version = "=2.0.0" }
+solana-spl-ids = { path = "spl-ids", version = "=2.0.0" }
```

Comment:
Can you please put this in the correct place A-Z? Should be after `solana-send-transaction-service`

Reply:
should be fixed